<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serverSettingWgt</class>
 <widget class="QWidget" name="serverSettingWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget{
    
	background-color: #eeeeee;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px;
}
QLabel{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#868989;
line-height:12px;
text-align:right;
}
QCheckBox{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#868989;
line-height:12px;
text-align:right;
}
QPushButton{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#323434;
letter-spacing:0;
line-height:12px;
text-align:center;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget{
    
	background-color: #eeeeee;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="settingsPage">
         <property name="styleSheet">
          <string notr="true">#settingsPage{
    
	background-color: #eeeeee;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>90</number>
          </property>
          <property name="topMargin">
           <number>61</number>
          </property>
          <property name="rightMargin">
           <number>130</number>
          </property>
          <property name="bottomMargin">
           <number>41</number>
          </property>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>9</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>9</number>
            </property>
            <property name="horizontalSpacing">
             <number>10</number>
            </property>
            <property name="verticalSpacing">
             <number>14</number>
            </property>
            <item row="1" column="1" colspan="2">
             <widget class="QLineEdit" name="portLineEdit">
              <property name="minimumSize">
               <size>
                <width>206</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>206</width>
                <height>27</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#portLineEdit{
background:#ffffff;
border:1px solid #c3caca;
border-radius:2px;
padding-left:9px;
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#323434;
letter-spacing:1px;
text-align:justify;
}</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QPushButton" name="cancelBtn">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>26</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#cancelBtn{
background:#ffffff;
border-radius:2px;
}
#cancelBtn:pressed{
margin:1px;
}
</string>
              </property>
              <property name="text">
               <string>取消</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="2">
             <widget class="QIPLineEdit" name="ipLineEdit">
              <property name="minimumSize">
               <size>
                <width>206</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>206</width>
                <height>27</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#ipLineEdit{
background:#ffffff;
border:1px solid #c3caca;
border-radius:2px;
}

QLineEdit{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#323434;
letter-spacing:1px;
text-align:justify;
}
</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QPushButton" name="confirmBtn">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>26</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#confirmBtn{
background:#ffd500;
border-radius:2px;
}
#confirmBtn:pressed{
margin:1px;
}
</string>
              </property>
              <property name="text">
               <string>确认</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>服务器地址：</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>服务器端口：</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="warnLabel">
              <property name="styleSheet">
               <string notr="true">#warnLabel{   
color: rgb(255, 0, 0);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tipPage">
         <property name="styleSheet">
          <string notr="true">#tipPage{
	background-color: #eeeeee;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>110</number>
          </property>
          <property name="topMargin">
           <number>50</number>
          </property>
          <property name="rightMargin">
           <number>110</number>
          </property>
          <property name="bottomMargin">
           <number>60</number>
          </property>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <property name="horizontalSpacing">
             <number>20</number>
            </property>
            <property name="verticalSpacing">
             <number>10</number>
            </property>
            <item row="2" column="1">
             <widget class="QPushButton" name="modifyBtn">
              <property name="minimumSize">
               <size>
                <width>110</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>110</width>
                <height>32</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#modifyBtn{
background:#ffd500;
border-radius:2px;
}
#modifyBtn:pressed{
margin:1px;
}</string>
              </property>
              <property name="text">
               <string>修改服务器地址</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1" colspan="3">
             <widget class="QLabel" name="tipLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font-size:14px;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="reconnectBtn">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>32</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#reconnectBtn{
background:#ffffff;
border-radius:2px;
}
#reconnectBtn:pressed{
margin:1px;
}</string>
              </property>
              <property name="text">
               <string>重连</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="3">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="progressPage">
         <property name="styleSheet">
          <string notr="true">#progressPage{
	background-color: #eeeeee;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px;
}</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="2">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>82</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>146</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="imageLabel">
            <property name="minimumSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="hintLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>连接服务器中...</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>146</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>81</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QIPLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qiplineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
